packages/cosmos/buildinfo.json

```diff
@@ -5,8 +5,8 @@
   ],
   "single_source": {
     "kind": "url",
-    "url": "https://downloads.dcos.io/cosmos/0.5.1-18/cosmos-server-0.5.1-18-one-jar.jar",
-    "sha1": "49f855f2618deace531fb874a796568f0140bf1c"
+    "url": "https://downloads.dcos.io/cosmos/0.5.3-21/cosmos-server-0.5.3-21-one-jar.jar",
+    "sha1": "b89be2021a46664f8ebe413846e44e89dcf5564a"
```

Comment:
Hi @takirala  - Will there be any admin-router test cases or dcos-integration tests cases applicable with this change?

When I read " When running uninstall on a service that doesn't have any environment variable configured in Marathon, it fails with the following exception."

It seems to me that we can have an integration test case with and env variable, try to install and uninstall the pacakge and assert the uninstallation does not fail.

Please let me know if this is covered else, if it is not covered, we could either cover it or add as TODO to bring in a different PR.

Reply:
Thanks @orsenthil 

Created [DCOS_OSS-2221](https://jira.mesosphere.com/browse/DCOS_OSS-2221) to address this in a future pr.

We have significant number of tests covering other aspects of this functionality [here](https://github.com/dcos/cosmos/blob/master/cosmos-integration-tests/src/main/scala/com/mesosphere/cosmos/handler/UninstallHandlerSpec.scala) and I think the reason this test was missed (or not written) in first place it feels like presence `env` is minimum guarantee of the contract from outside of cosmos. Nonetheless, I have created a ticket for that. I don't know if it is straightforward to write an integration test yet, if not, we can have a unit test at the least. 
/cc @jsancio correct me if I am wrong.

Reply:
Thanks @takirala  - that sounds good to me.